runtime/src/iree/io/formats/gguf/gguf_parser.c

```diff
@@ -671,11 +672,11 @@ static iree_status_t iree_io_parse_gguf_index_from_memory(
   }
   uint32_t version = 0;
   IREE_RETURN_IF_ERROR(iree_io_gguf_parse_uint32(&contents, &version));
-  if (version != GGUF_VERSION) {
+  if (version < GGUF_MIN_VERSION || version > GGUF_MAX_VERSION) {
     return iree_make_status(
         IREE_STATUS_UNIMPLEMENTED,
-        "GGUF format version %u is unsupported; expected version %u", version,
-        GGUF_VERSION);
+        "GGUF format version %u is unsupported; expected %u < version < %u",
```

Comment:
```suggestion
        "GGUF format version %u is unsupported; expected %u <= version <= %u",
```
if you want the equality, otherwise this reads `2 < version < 3` which is odd - alternatively can just say `expected version %u-%u`